data-and-functions-1/src/functions/getProductById.js

```diff
@@ -0,0 +1,11 @@
+const getProductById = (data, id) => {
+  if (data == null || data.products == null || id == null) {
+    return null
+  }else{
+    return data.products.find((products) => {
```

Comment:
Change the name of the `find` function parameter to reflect the fact that it's just a single product. See my comments below.